src/Collectors/Contracts/ThinkOfBetterNameInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+declare(strict_types=1);
+
+namespace Miquido\RequestDataCollector\Collectors\Contracts;
+
+interface ThinkOfBetterNameInterface
```

Comment:
weird Interface naming...

Reply:
Just temporary name for the time this PR is a draft :)